packages/arb-validator/nodegraph/stakedNodeGraph.go

```diff
@@ -114,8 +113,9 @@ func (sng *StakedNodeGraph) Equals(s2 *StakedNodeGraph) bool {
 func (sng *StakedNodeGraph) CreateStake(ev arbbridge.StakeCreatedEvent) {
 	nd, ok := sng.nodeFromHash[ev.NodeHash]
 	if !ok {
-		log.Println("Bad location", ev.NodeHash)
-		panic("Tried to create stake on bad node")
+		logger.Panic().
```

Comment:
We should probably use `Fatal` here instead of panic for consistency. I don't actually know the different if any between them